app/styles/_markdown.scss

```diff
@@ -0,0 +1,3 @@
+.markdown-body--empty {
+    color: $text--light;
+}
```

Comment:
You can remove this file and rebase on `upstream/develop` which will pull this file in and add the class there. I forgot you were not rebased yet onto our `upstream/develop`.